dspace-spring-rest/src/main/java/org/dspace/app/rest/utils/Utils.java

```diff
@@ -219,4 +236,60 @@ public static String getFileName(MultipartFile multipartFile)
             return multipartFile.getName();
         }
     }
+
+    private List<DSpaceObject> constructDSpaceObjectList(Context context,
```

Comment:
A very important method for uri-list.  Could we comment this one please?

Reply:
The JavaDocs have been included